app.py

```diff
@@ -114,26 +117,37 @@ def main(arguments):
         # 3. store new model
 
         # check to see if new model should be created
-        cursor.execute("SELECT * FROM ratings")
-        current_ratings_length = cursor.rowcount
+        # select the maximum time stamp from the ratings database
+        cursor.execute("""
+            SELECT timestamp FROM ratings ORDER BY timestamp DESC LIMIT 1;
```

Comment:
do we need to be concerned about the carriage returns in here? (ie since it's a multi-line string)